docker/build-rpm-in-docker.sh

```diff
@@ -1,6 +1,48 @@
 #!/bin/sh
-cd $(dirname $0)
-mkdir -p build-dir-$1
-cp ../* build-dir-$1
-cp -r ../dist build-dir-$1/dist
-sudo docker run -i -t -v ${PWD}/build-dir-$1:/build/build-dir-$1 erlang-rpm-build-$1
+#
+#
+#
+centos_version="$1"
+build_dir="build-dir-$centos_version"
+packages=( "Makefile" "*.patch" "erlang.spec" "Erlang_ASL2_LICENSE.txt" )
+#
+#
+#
+if [ -z $centos_version ]
+then
+	echo "
+Ops: parameters error
+first: version, ex: 6 or 7
+-----------------------------------------
+Ex: ./build-rpm-in-docker.sh 7
+"
+	exit 1
+fi
+#
+#
+#
+if [ -e "$build_dir" ]
+then
+	rm -rf "$build_dir"
```

Comment:
here you have to use `sudo`, since `sudo` writes the files  inside `$build_dir` 

should be:
```
sudo rm -rf "$build_dir"
```